go/test/endtoend/recovery/pitr/binlog_server.go

```diff
@@ -38,7 +38,7 @@ type mysqlMaster struct {
 
 // newBinlogServer returns an instance of binlog server
 func newBinlogServer(hostname string, port int) (*binLogServer, error) {
-	dataDir := path.Join(os.Getenv("VTDATAROOT"), binlogDataDir)
+	dataDir := path.Join(os.Getenv("VTDATAROOT"), binlogDataDir+"_"+fmt.Sprintf("%d", port))
```

Comment:
nit: if we are doing fmt.Sprintf, can't the whole string be created this way instead of mixing concatenation and sprintf?

Reply:
Used fmt.Sprintf() func to create a string